dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.h

```diff
@@ -227,6 +231,8 @@ class SegmentReadTaskPool : private boost::noncopyable
     // The memory tracker of MPPTask.
     MemoryTrackerPtr mem_tracker;
 
+    std::once_flag add_to_scheduler;
```

Comment:
Is there any chance that a taskpool be added twice since we already do protection by `task_pool_added`?

Reply:
@flowbehappy there could be multiple `UnorderedInputStream` hold the same ptr to one `SegmentReadTaskPool `
https://github.com/pingcap/tiflash/blob/master/dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp#L1003-L1014

Reply:
Add some comments to explain the var here?